### PR TITLE
Updates for Orange and Sosh (FR)

### PR DESCRIPTION
**Pull request by marc78280**
2FA by "Mobile Connect" exists since a few years (2019 ?)

### CODE REVIEW

**Comment by Carlgo11** [comment]
Hello @marc78280, thanks for the pull request!
I see you've added a lot of websites to this pull request. In the future, we would very much prefer it if you limited each pull request to just one website entry, as it would be easier for us to sort through.

File: entries/c/caisse-epargne.fr.json [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
-      "language": "fr"
-    },
+    "tfa": [
+      "sms",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Sécur’Pass"
+    ],
+    "documentation": "https://www.caisse-epargne.fr/comptes-cartes/dsp2/#anchor-authentification-forte",
```
Comment (Carlgo11) [inline entries/c/caisse-epargne.fr.json:12]: What's described sounds more like authorization for specific user actions than authentication at the login phase.
We distinguish between the two in the section about authorization in [Contributing.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#authorization).
Please confirm if this is the case or if 2FA is also prompted on login.

File: entries/f/fortuneo.fr.json [hunk 2]
```diff
@@ -2,11 +2,10 @@
...
     "url": "https://www.fortuneo.fr/",
-    "contact": {
-      "facebook": "fortuneo.banque",
-      "twitter": "fortuneo",
-      "language": "fr"
-    },
+        "tfa": [
+      "sms"
+    ],
+    "documentation": "https://www.fortuneo.fr/securite/authentification-code-securite",
```
Comment (Carlgo11) [inline entries/f/fortuneo.fr.json:8]:
```suggestion
    "tfa": [
      "sms"
    ],
    "documentation": "https://www.fortuneo.fr/securite/authentification-code-securite",
```

File: entries/i/impots.gouv.fr.json [hunk 3]
```diff
@@ -2,16 +2,15 @@
...
     "url": "https://www.impots.gouv.fr",
-    "contact": {
-      "facebook": "Direction-générale-des-Finances-publiques-250996308359760",
-      "twitter": "dgfip_officiel",
-      "language": "fr"
-    },
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://www.impots.gouv.fr/particulier/questions/comment-renforcer-la-securite-de-lacces-mon-espace-particulier",
```
Comment (Carlgo11) [inline entries/i/impots.gouv.fr.json:8]: What's described sounds more like authorization for specific user actions than authentication at the login phase.
We distinguish between the two in the section about authorization in [Contributing.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#authorization).
Please confirm if this is the case or if 2FA is also prompted on login.

File: entries/o/orange.fr.json [hunk 4]
```diff
@@ -2,16 +2,20 @@
   "Orange": {
     "domain": "orange.fr",
     "url": "https://www.orange.fr/portail",
-    "contact": {
-      "facebook": "Orange.France",
-      "twitter": "Orange_conseil",
-      "language": "fr"
-    },
+    "tfa": [
+          "custom-hardware"
```
Comment (Carlgo11) [inline entries/o/orange.fr.json:6]:
```suggestion
      "custom-hardware"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/c/caisse-epargne.fr.json (hunk 1 above)
Comment (marc78280) [reply entries/c/caisse-epargne.fr.json:12]: I confirm sadly that french banks only use 2FA for authorization and not for each logiin with account/password

File: entries/i/impots.gouv.fr.json (hunk 3 above)
Comment (marc78280) [reply entries/i/impots.gouv.fr.json:8]: I confirm you are right, sorry Carl

Comment (Carlgo11) [reply entries/i/impots.gouv.fr.json:8]: pas de soucis! 🙂
You've changed the following sites:
* caisse-epargne.fr
* fortuneo.fr
* impots.gouv.fr
* labanquepostale.fr
* lcl.fr
* orange.fr
* sosh.fr

Which of these sites support two-factor authentication and not just two-step authorization?

**Pull request closed without merging**